src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxTcpConnectionInformation.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    // This class is not Linux-specific. It is just a data container.
+    // TODO: Combine this with SystemTcpConnection. Make Windows construct that object differently.
```

Comment:
I actually had meant that we could consider just using the same type for all platforms, because it's just a data container. We'd have to rework how the SystemTcpConnectionInformation constructors work, though (they read the data in from Win32 structs, rather than having the data passed into the constructor).